ezbake/config/conf.d/auth.conf

```diff
@@ -86,6 +86,16 @@ authorization: {
             sort-order: 500
             name: "puppetlabs cert statuses"
         },
+        {
+            match-request: {
+                path: "/puppet-ca/v1/ca-expirations"
+                type: path
+                method: get
+            }
+            allow: "*"
```

Comment:
what made you choose this, as opposed to, like, what we allow for the `certificate_statuses` endpoint above?

Reply:
That endpoint is generally used to change the state of certificates (e.g. to sign or revoke them). This endpoint is read-only, so I figured it didn't need to be as secured. The one above allows _only_ the master to query it, which isn't particularly useful here. This one requires someone to have puppet certs, but anyone with puppet certs can read it, for example, the console.

In PE, we could lock it down tighter and specifically allow the console only. But in FOSS we don't have that option, so this is a good compromise.

If someone thinks we should do something different, please let me know.

Reply:
That makes sense to me, thanks for explaining!

Reply:
Filed https://tickets.puppetlabs.com/browse/PE-29718 for adding the auth rules in PE. Appreciate thoughts on the question in there about whether to allow only the console.